src/components/sections/moving-border.tsx

```diff
@@ -0,0 +1,139 @@
+"use client";
+import React from "react";
+import {
+  motion,
```

Comment:
Will need to switch to m instead of motion since we are using reducing bundle size